.travis.yml

```diff
@@ -429,7 +429,9 @@ jobs:
     name: Publishing current Git tagged version of dist to PyPI
     if: repo == "ansible/molecule" AND tag IS present
     env:
-      TOXENV: metadata-validation
+      TOXENV: metadata-validation,build-dists
+    before_deploy:
+    - echo > setup.py
```

Comment:
FTR: deployment provider calls `setup.py` no matter what, but we build things manually so I'm wiping its contents and it will not affect pre-created dists this way but will still use twine to upload things to PyPI :)

(also a battle-tested hack)